src/NServiceBus.Transport.SQS/TransportMessage.cs

```diff
@@ -28,9 +29,9 @@ public TransportMessage(OutgoingMessage outgoingMessage, DispatchProperties prop
             }
 
 #if NETFRAMEWORK
-            Body = outgoingMessage.Body.Length != 0 ? Convert.ToBase64String(outgoingMessage.Body.ToArray()) : "empty message";
+            Body = outgoingMessage.Body.Length != 0 ? Convert.ToBase64String(outgoingMessage.Body.ToArray()) : EmptyMessage;
 #else
-            Body = outgoingMessage.Body.Length != 0 ? Convert.ToBase64String(outgoingMessage.Body.Span) : "empty message";
+            Body = outgoingMessage.Body.Length != 0 ? Convert.ToBase64String(outgoingMessage.Body.Span) : EmptyMessage;
 #endif
```

Comment:
In the case of native integration, is there a chance that `Body` is `null`, or will it always be an empty string??

Reply:
I have added a test for null.